build/travis/job2_AppImage/build.sh

```diff
@@ -37,17 +37,35 @@ else
   makefile_overrides="" # use Makefile defaults
 fi
 
-# Build AppImage. Are we cross-compiling?
+# Build AppImage depending on arch specified in $1 if cross-compiling, else default build x86_64
 case "$1" in
-  --32bit )
+
+  --armhf )
+    shift
+    # build MuseScore inside debian x86-64 multiarch image containing arm cross toolchain and libraries
+    docker run -i -v "${PWD}:/MuseScore" \
+      ericfont/musescore:jessie-crosscompile-armhf \
+      /bin/bash -c \
+      "/MuseScore/build/Linux+BSD/portable/RecipeArm --build-only $makefile_overrides"
```

Comment:
ok.  well I only want to run test for the arm build.  I can add the --test parameter, but I'll only add it for arm.  I'll do test after upload, since could take some time if being emulated.